### DISCUSSION TITLE
What is the hardware configuration for linux VMs?

### DISCUSSION THREAD

**Discussion by bob-ashforth-sp** [actions/virtual-environments#5076] [Image configuration]
I would like to use multiple cores to perform parallel tests, but can't find any mention of how many cores are configured for ubuntu runners. Can anyone provide this information?

**Comment by al-cheb** [comment]
Currently, we use [`Standard_DS2_v2`](https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series) vm size with spec - (2 vcpu / 7gb).

```
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   46 bits physical, 48 bits virtual
CPU(s):                          2
```

**Reply by catthehacker** [reply]
It's also mentioned in [here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#cloud-hosts-for-github-hosted-runners) and [here](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)
